src/session_test.cpp

```diff
@@ -219,6 +219,30 @@ TEST_F(SessionTest, RequestResponseSuccess) {
   ASSERT_FALSE(got.response.o2.has_value());
 }
 
+TEST_F(SessionTest, BreakPointRequestResponseSuccess) {
+  server->registerHandler(
+      [&](const dap::SetBreakpointsRequest&) {
+    dap::SetBreakpointsResponse rsp;
+    dap::array<dap::Breakpoint> breakpoints;
+    dap::Breakpoint bpItem;
+    bpItem.line = 2;
+    breakpoints.emplace_back(std::move(bpItem));
+    rsp.breakpoints.swap(breakpoints);
+    return rsp;
+  });
+
+  bind();
+
+  dap::SetBreakpointsRequest request;
+  auto response = client->send(request);
+
+  auto got = response.get();
+  
+  // Check response was received correctly.
+  ASSERT_EQ(got.error, false);
+  ASSERT_EQ(got.response.breakpoints.size(), 1);
```

Comment:
Presubmits failing due to unsigned / signed comparison. Please change `1` to `1U`